drupal/rootfs/etc/islandora/utilities.sh

```diff
@@ -284,6 +284,43 @@ function fedora_url {
     fi
 }
 
+
+# Allow modifications to settings.php by changing ownership and perms
+function allow_settings_modifications {
+    local site="${1}"; shift
+    local drupal_root=$(drush drupal:directory)
+    local subdir=$(drupal_site_env "${site}" "SUBDIR")
+    local site_directory=$(realpath "${drupal_root}/sites/${subdir}")
+
+    # send debug output to stderr because the caller typically captures output from this function.
+    #>&2 echo "adjusting ownership of ${site_directory}/settings.php"  
+    if [ -f "${site_directory}/settings.php" ]; then
+        previous_owner_group=$(stat -c "%u:%g" "${site_directory}/settings.php")
+        chown 100:101 "${site_directory}/settings.php"
+        chmod a=rwx "${site_directory}/settings.php"
+    fi
+    if [ ! -z "${previous_owner_group}" ]; then
+        echo ${previous_owner_group}
+    fi
+}
+
+# Restore ownership of settings.php so that it is readable/writable outside of docker
+function restore_settings_ownership {
+    local site="${1}"; shift
+    local previous_owner_group="${1}"; shift
+    local drupal_root=$(drush drupal:directory)
+    local subdir=$(drupal_site_env "${site}" "SUBDIR")
+    local site_directory=$(realpath "${drupal_root}/sites/${subdir}")
+
+    # Restore owner/group to previous value
+    if [ ! -z "${previous_owner_group}" ]; then
+        #echo "restoring ${site_directory}/settings.php"
+        #echo "previous_owner_group ${previous_owner_group}"
```

Comment:
We can remove the commented out `#echo` statements.

Reply:
Agreed - I've taken them out.